Description: Fix "missing field `deadline_nsec` in initializer of `drm_sys::drm_syncobj_*`"
 Some autopkgtests raise the error in description. We backport the patch from v0.8.0.
Origin: upstream, https://github.com/Smithay/drm-rs.git
Applied-Upstream: 75cd9d3c86f5209a9da955a0eb58fbab882fa3e1
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/syncobj.rs
+++ b/src/syncobj.rs
@@ -110,6 +110,7 @@
         },
         first_signaled: 0,
         pad: 0,
+        deadline_nsec: 0,
     };
 
     unsafe {
@@ -181,6 +182,7 @@
         },
         first_signaled: 0,
         pad: 0,
+        deadline_nsec: 0,
     };
 
     unsafe {
